src/components/Menu/hooks/useCompetitionStatus.ts

```diff
@@ -0,0 +1,39 @@
+import { useMemo, useEffect, useState } from 'react'
+import {
+  SmartContractPhases,
+  CompetitionPhases,
+  LIVE,
+  FINISHED,
+  CLAIM,
+  OVER,
+} from 'config/constants/trading-competition/phases'
+import { useTradingCompetitionContractMoD } from 'hooks/useContract'
+
+export const useCompetitionStatus = () => {
+  const tradingCompetitionContract = useTradingCompetitionContractMoD(false)
+  const [currentPhase, setCurrentPhase] = useState(CompetitionPhases.REGISTRATION)
+
+  useEffect(() => {
+    const fetchCompetitionInfoContract = async () => {
```

Comment:
@ChefMomota Imo We should use immutable swr here, otherwise this will make 2 requests instead of one (one for menu status one for banner).

Reply:
@memoyil  Updated.